docs/en-us/dev/user_doc/dev_quick_start.md

```diff
@@ -0,0 +1,60 @@
+Note: Only for standalone development debugging use, the default use of H2 Database, Zookeeper Testing Server.
```

Comment:
"The default option is H2 Database and Zookeeper Testing Server" would be more natural

Reply:
Thanks for reviewing, I have updated it.